builtin/game/privileges.lua

```diff
@@ -90,3 +90,7 @@ core.register_privilege("debug", {
 	description = "Allows enabling various debug options that may affect gameplay",
 	give_to_singleplayer = false,
 })
+core.register_privilege("clearinv", {
+	description = "Can use clear inventory command",
```

Comment:
for other players

Also, maybe rename to make it even more obvious it's not for your own inv